### DISCUSSION TITLE
Documentation Question Regarding "Plugins can be included individually (using Bootstrap’s individual js/dist/*.js)"

### DISCUSSION THREAD

**Discussion by wyattnolen** [twbs/bootstrap#40536] [Q&A] [answered]
I was hoping to customize the JS I use to exclude javascript for things like Carousel, maybe Scrollspy, and other features I know I wont need. The documentation seems to indicate this possibility, but when downloading bootstrap, I do not see the js/dist directory. Am I just misunderstanding the documentation here? 

"Plugins can be included individually (using Bootstrap’s individual js/dist/*.js), or all at once using bootstrap.js or the minified bootstrap.min.js (don’t include both)."

Source: https://getbootstrap.com/docs/5.3/getting-started/javascript/

**Comment by julien-deramond** [comment]
Hello @wyattnolen 

How do you install Bootstrap?

With NPM, for instance, you can:
```
ju ߷ /tmp ➤ npm i bootstrap@latest
ju ߷ /tmp ➤ ls node_modules/bootstrap/js/dist/*.js                                                    
node_modules/bootstrap/js/dist/alert.js           node_modules/bootstrap/js/dist/offcanvas.js
node_modules/bootstrap/js/dist/base-component.js  node_modules/bootstrap/js/dist/popover.js
node_modules/bootstrap/js/dist/button.js          node_modules/bootstrap/js/dist/scrollspy.js
node_modules/bootstrap/js/dist/carousel.js        node_modules/bootstrap/js/dist/tab.js
node_modules/bootstrap/js/dist/collapse.js        node_modules/bootstrap/js/dist/toast.js
node_modules/bootstrap/js/dist/dropdown.js        node_modules/bootstrap/js/dist/tooltip.js
node_modules/bootstrap/js/dist/modal.js
```

**Reply by wyattnolen** [reply]
For this use case, no it wouldn't be NPM. Client does everything in their platform, no build process. Was hoping to download the files from the bootstrap website.

**Reply by julien-deramond** [reply, marked as answer]
OK, so you should have the same files by downloading the sources from https://getbootstrap.com/docs/5.3/getting-started/download/#source-files, in the `bootstrap-5.3.3/js/dist/` directory.

**Reply by wyattnolen** [reply]
Yes, that is perfect, thank you!
